docs/assets/includes/nav-multistep.md

```diff
@@ -0,0 +1,20 @@
+<nav class="nav-multistep">
```

Comment:
I suggest giving a more specific name to this "include file", to take into account that we may have more guided paths like this one in the future. For example, `nav-multistep-quickstart.md` or `nav-repository-onboarding.md`.

Reply:
Agreed. I updated the name to `nav-multistep-quickstart.md`.